projects/storefrontlib/cms-components/navigation/navigation/navigation-ui.component.ts

```diff
@@ -396,4 +396,18 @@ export class NavigationUIComponent implements OnInit, OnDestroy {
     }
     return depth > 0 && !node?.children ? -1 : 0;
   }
+
+  /**
+   * Returns the ID for the `aria-describedby` attribute of a button.
+   */
+  getAriaDescribedbyOfButton(node: NavigationNode): string | null {
+    return node.title === 'My Account' ? 'greeting' : null;
```

Comment:
Hm, it doesn't seem right to use the title as an identifier of the node. If this aria described by is used only for my account navigation, maybe we can pass down some input from `my-account-v2-navigation.component.ts` where `<cx-navigation-ui>` is used. Thats just from the top of my head. 
I'm not sure if using 'My Account' would work with i18n...